backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -144,6 +147,16 @@ def set_store
         redirect_to store_admin_order_url(@order)
       end
 
+      def set_channel
+        if @order.update_attributes(channel: params[:order][:channel])
```

Comment:
Why can't we update a channel along with the order?(e.g via `update`)

Reply:
because of strong params, if there is mass-assignment than it just throws an error, as i understand update not used right now, and we need to discuss it with @damianlegawiec .

Reply:
@damianlegawiec 
Please shine some light on this issue
Can we just include `channel` in the list of permitted attributes and use `update` instead?

Reply:
@damianlegawiec ^^ when you have a minute, please.

Reply:
@damianlegawiec ☝️